packages/ember-testing/lib/helpers/find.js

```diff
@@ -29,6 +30,15 @@ import { get } from 'ember-metal';
 export default function find(app, selector, context) {
   let $el;
   context = context || get(app, 'rootElement');
-  $el = app.$(selector, context);
-  return $el;
+  if (jQueryDisabled) {
```

Comment:
So to allow this to work in all cases, `find` needs to accept an element or selector...not just a selector.